docs/ops_guide/repush.md

```diff
@@ -2,7 +2,6 @@
 layout: default
 title: Repush
 parent: Operator Guides
-permalink: docs/ops_guide/repush
```

Comment:
Should we (could we?) also keep the permalinks? They look a little bit nicer. If I shared a permalink (e.g., [here](https://twitter.com/felixgv/status/1576981114866270208?s=21)) then it'll be broken after this change, right? Not a big deal, but just wondering.

Reply:
Yeah, that's true. The link will break in its current form. We can keep the permalinks if we standardize on a format

Reply:
I keep going back and forth in my head b/w having permalinks or not. I think I agree that having permalinks is a better idea. Updated the permalinks